plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -583,8 +583,6 @@ func ClusterRoles() []rbacv1.ClusterRole {
 			rbacv1helpers.NewRule(Read...).Groups(resourceGroup).Resources("deviceclasses").RuleOrDie(),
 			rbacv1helpers.NewRule(ReadUpdate...).Groups(resourceGroup).Resources("resourceclaims").RuleOrDie(),
 			rbacv1helpers.NewRule(ReadUpdate...).Groups(resourceGroup).Resources("resourceclaims/status").RuleOrDie(),
-			rbacv1helpers.NewRule(ReadWrite...).Groups(resourceGroup).Resources("podschedulingcontexts").RuleOrDie(),
-			rbacv1helpers.NewRule(Read...).Groups(resourceGroup).Resources("podschedulingcontexts/status").RuleOrDie(),
```

Comment:
Similar to above, cleaning `system:kube-scheduler` seems reasonable. Although I'd double check with folks from sig-auth for the best path forward.

Reply:
dropping no-longer needed things from a never-enabled-by-default role is fine